tests/suite/test_app_protect_integration.py

```diff
@@ -334,8 +334,12 @@ def test_ap_sec_logs_on(
         assert f'outcome="REJECTED"' in log_contents
 
         print("----------------------- Send valid request ----------------------")
+        headers = {
```

Comment:
is this change required for this PR? 🤔

Reply:
The new version of app protect introduced bot-signatures. Any typical non browser user-agent strings in the header (curl python etc.)  will be classified as "non-browser request" , so this particular test started failing. In order to make the request valid the user agent was set to one of a firefox browser.

Reply:
@rafwegv thanks for clarifying!

cc @vepatel so he's aware

Reply:
@pleshakov already had discussion with @rafwegv re the same.